cmd/templates/hook.tmpl

```diff
@@ -15,9 +15,9 @@ call_lefthook()
   if lefthook{{.Extension}} -h >/dev/null 2>&1
   then
     eval lefthook{{.Extension}} $@
-  elif test -f "$dir/node_modules/@arkweid/lefthook/bin/lefthook{{.Extension}}"
+  elif test -f "$dir/node_modules/@arkweid/lefthook/bin/index.js"
   then
-    eval "$dir/node_modules/@arkweid/lefthook/bin/lefthook{{.Extension}} $@"
+    eval "$dir/node_modules/@arkweid/lefthook/bin/index.js $@"
```

Comment:
This doesn't work. The test is always true

Reply:
You mean `test -f "$dir/node_modules/@arkweid/lefthook/bin/index.js"` always return `true`?

Reply:
Yes. There is no point in the test. Also, you can't eval JavaScript directly. In the new version, there is no need for `index.js`.

Reply:
`test -f "$dir/node_modules/@arkweid/lefthook/bin/index.js"` only returns true if `lefthook` is installed via npm/yarn
When uninstalled it would only echo `Can't find lefthook in PATH`
If `lefthook` installed via Homebrew, the `test -f` line won't even be called

<details>
<summary>My custom `pre-commit` file</summary>
```sh
#!/bin/sh

if [ "$LEFTHOOK" = "0" ]; then
  exit 0
fi

if [ -t 1 ] ; then
  exec < /dev/tty ; # <- enables interactive shell
fi

dir="$(git rev-parse --show-toplevel)"

call_lefthook()
{
  if lefthook -h >/dev/null 2>&1
  then
    eval lefthook $@
  elif test -f "$dir/node_modules/@arkweid/lefthook/bin/index.js"
  then
    eval "$dir/node_modules/@arkweid/lefthook/bin/index.js $@"
  elif bundle exec lefthook -h >/dev/null 2>&1
  then
    bundle exec lefthook $@
  elif npx lefthook -h >/dev/null 2>&1
  then
    npx lefthook $@
  elif yarn lefthook -h >/dev/null 2>&1
  then
    yarn lefthook $@
  else
    echo "Can't find lefthook in PATH"
  fi
}



call_lefthook "run pre-commit $@"

```
</details>


<details>
<summary>My custom `pre-commit` file with `echo` on each "branch"</summary>
```sh
#!/bin/sh

if [ "$LEFTHOOK" = "0" ]; then
  exit 0
fi

if [ -t 1 ] ; then
  exec < /dev/tty ; # <- enables interactive shell
fi

dir="$(git rev-parse --show-toplevel)"

call_lefthook()
{
  if lefthook -h >/dev/null 2>&1
  then
    echo "pure lefthook route"
    eval lefthook $@
  elif test -f "$dir/node_modules/@arkweid/lefthook/bin/index.js"
  then
    echo "node_modules/@arkweid/lefthook/bin/index.js route"
    eval "$dir/node_modules/@arkweid/lefthook/bin/index.js $@"
  elif bundle exec lefthook -h >/dev/null 2>&1
  then
    echo "bundle exec route"
    bundle exec lefthook $@
  elif npx lefthook -h >/dev/null 2>&1
  then
    echo "npx route"
    npx lefthook $@
  elif yarn lefthook -h >/dev/null 2>&1
  then
    echo "yarn route"
    yarn lefthook $@
  else
    echo "Can't find lefthook in PATH"
  fi
}



call_lefthook "run pre-commit $@"

```
</details>

Reply:
You are saying `npx lefthook` is already using `node_modules/@arkweid/lefthook/bin/index.js`.
So this check is redundant and should be removed?

Reply:
Yes. This PR doesn't solve anything. 

index.js is used when running npx lefthook
https://github.com/evilmartians/lefthook/blob/6c84d855dbb7874d9a82713c15441af940851a44/.npm/package.json#L7

Reply:
I will update this PR to just remove this useless branch then

Something I just noticed:
In https://github.com/evilmartians/lefthook/blob/6c84d855dbb7874d9a82713c15441af940851a44/cmd/templates/hook.tmpl#L24-L29

I see `@arkweid/lefthook` and `lefthook` used at the same time
Is it supposed to use `@arkweid/lefthook` only?

[`lefthook`](https://www.npmjs.com/package/lefthook) != [`@arkweid/lefthook`](https://www.npmjs.com/package/@arkweid/lefthook) on npmjs.com